course_catalog/views.py

```diff
@@ -439,12 +440,26 @@ class PodcastViewSet(viewsets.ReadOnlyModelViewSet):
     permission_classes = (ReadOnly & PodcastFeatureFlag,)
 
     queryset = (
-        Podcast.objects.filter(published=True, episodes__published=True)
+        Podcast.objects.filter(published=True)
         .prefetch_related(
             Prefetch("offered_by", queryset=LearningResourceOfferor.objects.all()),
             Prefetch("topics", queryset=CourseTopic.objects.all()),
         )
-        .annotate(episode_count=Count("episodes"))
+        .annotate(
```

Comment:
Can you include a comment about why this somewhat complex query is needed here instead of just `Count("episodes")`?

Reply:
Sure, just added one

Reply:
Sorry I forgot to get to this earlier? Can we just use conditional aggregation here if the only change is that we want to count published episodes only? https://docs.djangoproject.com/en/2.2/ref/models/conditional-expressions/#conditional-aggregation

I think it would look something like this:
```python
Podcast.objects.filter(published=True)
        .prefetch_related(	        .prefetch_related(
            Prefetch("offered_by", queryset=LearningResourceOfferor.objects.all()),	            Prefetch("offered_by", queryset=LearningResourceOfferor.objects.all()),
            Prefetch("topics", queryset=CourseTopic.objects.all()),	            Prefetch("topics", queryset=CourseTopic.objects.all()),
        )	        )
        .annotate(
            episode_count=Count("episodes", filter=Q(is_published=True))
        )
```